test/e2e/infra/vagrant/push_flow_aggregator.sh

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env bash
+
+function usage() {
+    echo "Usage: push_flow_aggregator.sh [-fc|--flow-collector] [-h|--help]
+    Push the latest flow-aggregator image to all vagrant nodes and restart the service
+          --flow-collector   Flow collector is the externalFlowCollectorAddr configMap parameter
+                             It show given in format IP:port:proto. Example: 192.168.1.100:4739:udp"
+}
+
+# Process execution flags
+FLOW_COLLECTOR=""
+
+while [[ $# -gt 0 ]]
+do
+key="$1"
+
+case $key in
+-fc|--flow-collector)
+FLOW_COLLECTOR="$2"
+shift 2
+;;
+-h|--help)
+usage
+exit 0
+;;
+*)
+usage
+exit 1
+esac
+done
+
+if [ "$FLOW_COLLECTOR" == "" ]; then
+    echoerr "--flow-collector must be provided in the expected address format"
+    usage
+    exit 1
+fi
+
+: "${NUM_WORKERS:=1}"
+SAVED_IMG=/tmp/flow-aggregator.tar
+IMG_NAME=antrea/flow-aggregator:latest
```

Comment:
image name will be updated to projects.registry.vmware.com/antrea/antrea-ubuntu:latest

Reply:
Yes, this seems like a recent change to support pulled images along with compiled images.

Reply:
Just realized we need to make the change for flow aggregator too to support tag with "project.registry..."